### PR TITLE
Derive list of countries from JDK

### PR DESCRIPTION
**Pull request by vkolomeyko**

### CODE REVIEW

File: core/src/main/kotlin/net/corda/core/internal/CountryCode.kt [hunk 1]
```diff
@@ -1,255 +1,6 @@
...
-        "VG",
-        "VI",
-        "WF",
-        "EH",
-        "YE",
-        "ZM",
-        "ZW"
-)
\ No newline at end of file
+val countryCodes: Set<String> = ImmutableSet.copyOf(Locale.getISOCountries())
```
Comment (shamsasari) [inline core/src/main/kotlin/net/corda/core/internal/CountryCode.kt]: Very good. There's now no need for this class. Move `countryCodes` into `CordaX500Name`'s companion object make it private.

Comment (vkolomeyko) [reply core/src/main/kotlin/net/corda/core/internal/CountryCode.kt]: Done

**Comment by patrickkuo** [comment]
one worry I have is if this list ever change, the validation will be different depends on which JRE corda is running on.... but I guess changing county code is a very rare case.

**Comment by shamsasari** [comment]
We're going to be linking specific versions of Java to Corda platform version, so if this list gets updated it will only happen by us upgrading the JRE version.

**Pull request merged**